jolt-sdk/macros/src/lib.rs

```diff
@@ -443,6 +446,18 @@ impl MacroBuilder {
         }
     }
 
+    fn make_write_to_file(&self) -> TokenStream2 {
```

Comment:
I'm not a huge fan of using an env variable here; I'd rather just call `save_to_file` directly from the host

Reply:
The issue I was hoping to address with the env var is in regards to saving the elf file when it is compiled to a temporary directory at run time. Since the program isn't directly accessible in the host I thought this was the least intrusive. Open to suggestions.